package.json

```
@@ -11,7 +11,7 @@
     "karma": "cross-env NODE_ENV=test babel-node $(npm bin)/karma start",
     "test": "if [ ! $SKIP_TEST ]; then npm run lint && npm run flow-typed && npm run flow && npm run jest-ssr && npm run karma && npm run jest-screenshot; fi;",
     "webpack": "babel-node $(npm bin)/webpack --progress",
-    "build": "npm run test && npm run webpack",
+    "build": "npm run webpack",
```

Let's make sure to add the tests back to the `build` task.
```suggestion
    "build": "npm run test && npm run webpack",
```

---

reverted back